components/specification/publish

```diff
@@ -153,18 +153,23 @@ contains() {
 }
 
 devmarker="dev"
-root=published
-
+root=${1:-published}
+root=${root%/}
 
 # Clean up
 
-rm -rf "$root"
+rm -rf "$root/*"
```

Comment:
As mentioned in the PR description, the idea is to generate the output of this script into a Git repository to have diffs. In this case, we do not want to remove the `.git` subfolder minimally - maybe the existing `Documentation` subfolder in the mid-term. Arguably, this cleanup step could be moved outside the `publish` script which is really a build target and people could go with the following:

```
rm -rf target_path & ./publish target_path
```